src/h2load.cc

```diff
@@ -352,6 +414,16 @@ Client::Client(uint32_t id, Worker *worker, size_t req_todo)
 
   ev_timer_init(&request_timeout_watcher, client_request_timeout_cb, 0., 0.);
   request_timeout_watcher.data = this;
+
+  if (worker->config->warm_up_time > 0) {
+    ev_timer_init(&duration_watcher, duration_timeout_cb, 
+                  worker->config->rate_period, 0.);
+    duration_watcher.data = this;
+    
+    ev_timer_init(&warmup_watcher, warmup_timeout_cb, 
```

Comment:
Okay. I have changed it.
But right now, warmup time is separate from main duration. so `--warm-up-time 5 --duration 10` means that it will wait 5 seconds for warming up and then 10 seconds of measurement.